### ISSUE TITLE
Can't move to first slide with moveToSlide

### ISSUE THREAD

**Issue by ejahn** [sheikalthaf/ngx-carousel#55]
Hi @sheikalthaf,

When changing the value for the input moveToSlide, jumping to the first slide with index 0 doesn't work.

I assume this is because 0 gets evaluated as falsy when using logical operators.
In ngx-carousel.components.ts, line 253:

``` typescript
ngOnChanges(changes: SimpleChanges) {
  // tslint:disable-next-line:no-unused-expression
  this.moveToSlide &&
    !changes.moveToSlide.firstChange &&
    this.moveTo(changes.moveToSlide.currentValue);
}
```

Changing line 255 to: `this.moveToSlide != null &&` would solve the problem.

Did you have something special in mind, when writing this code, or is this maybe a bug?

**Comment by iamlothian** [comment]
@ejahn good spot. Some propper testing should have picked this up. For now setting to `-1` works for my usecase.

**Comment by sheikalthaf** [comment]
@ejahn   Fixed the bug in Build v1.3.4
Big thanks to @iamlothian for the solution
Thanks
